builtin/mainmenu/dlg_settings_advanced.lua

```diff
@@ -632,11 +632,11 @@ local function create_change_setting_formspec(dialogdata)
 		end
 		-- Flags
 		formspec = table.concat(fields)
-				.. "checkbox[0.5," .. height - 0.6 .. ";cb_defaults;defaults;" -- defaults
+				.. "checkbox[0.5," .. height - 0.6 .. ";cb_defaults;"..fgettext("defaults")..";" -- defaults
 				.. tostring(flags["defaults"] == true) .. "]" -- to get false if nil
-				.. "checkbox[5," .. height - 0.6 .. ";cb_eased;eased;" -- eased
+				.. "checkbox[5," .. height - 0.6 .. ";cb_eased;"..fgettext("eased")..";" -- eased
```

Comment:
Inconsistent use of spaces around '..'
Possible overly-long line (limit is 90 with tab size 4)